vivarium_convenience/processes/convenience_kinetics.py

```diff
@@ -288,18 +288,22 @@ def next_update(self, timestep, states):
                     # separate the state_id and port_id
                     if port_id in port_state_id:
                         state_id = port_state_id[1]
-                        state_flux = coeff * flux * timestep
+                        state_flux = (
+                            coeff * flux * timestep * units.millimolar)
```

Comment:
Maybe the units should be configurable? `units(self.parameters['flux_unit'])`?

Reply:
done